daemon/containerd/service.go

```diff
@@ -27,18 +28,19 @@ import (
 
 // ImageService implements daemon.ImageService
 type ImageService struct {
-	client              *containerd.Client
-	images              images.Store
-	content             content.Store
-	containers          container.Store
-	snapshotterServices map[string]snapshots.Snapshotter
-	snapshotter         string
-	registryHosts       docker.RegistryHosts
-	registryService     registryResolver
-	eventsService       *daemonevents.Events
-	pruneRunning        atomic.Bool
-	refCountMounter     snapshotter.Mounter
-	idMapping           idtools.IdentityMapping
+	client                  *containerd.Client
+	images                  images.Store
+	content                 content.Store
+	containers              container.Store
+	snapshotterServices     map[string]snapshots.Snapshotter
+	snapshotter             string
+	registryHosts           docker.RegistryHosts
+	registryService         registryResolver
+	eventsService           *daemonevents.Events
+	pruneRunning            atomic.Bool
+	refCountMounter         snapshotter.Mounter
+	idMapping               idtools.IdentityMapping
+	defaultPlatformOverride platforms.MatchComparer
```

Comment:
Can you add a comment that this is for testing-purposes only?

Perhaps add a whitespace in between (which also reduces the code-churn)

Reply:
Done